src/dotnet/APIView/APIViewWeb/Models/CommentThreadModel.cs

```diff
@@ -1,24 +1,29 @@
-﻿using System.Collections.Generic;
+﻿using Newtonsoft.Json;
+using System.Collections.Generic;
 using System.Linq;
 
 namespace APIViewWeb.Models
 {
     public class CommentThreadModel
     {
-        public CommentThreadModel(string reviewId, string lineId, IEnumerable<CommentModel> comments)
+        public CommentThreadModel(string reviewId, string lineId, IEnumerable<CommentModel> comments, string language)
         {
             ReviewId = reviewId;
             LineId = lineId;
             Comments = comments.Where(c => !c.IsResolve);
             var resolveComment = comments.FirstOrDefault(c => c.IsResolve);
             IsResolved = resolveComment != null;
             ResolvedBy = resolveComment?.Username;
+            Language = language;
         }
 
         public string ReviewId { get; set; }
         public IEnumerable<CommentModel> Comments { get; set; }
         public string LineId { get; set; }
         public bool IsResolved { get; set; }
         public string ResolvedBy { get; set; }
+
+        [JsonIgnore]
+        public string Language { get; set; }
```

Comment:
I don't think this belongs here. Review might have a language, a comment doesn't need one.